### PR TITLE
fix(php): remove & to be compatible with PHP 8

### PR DESCRIPTION
**Pull request by vargenau**
## Description

Fix old PHP syntax to be compatible with PHP 8.

### Changes

Remove ```&``` in ```array_push``` and before ```new```.

### CODE REVIEW

File: utils/freshmeat/Classes/FreshmeatRdfs.php [hunk 1]
```diff
@@ -153,7 +153,7 @@ public function XtractProjInfo($rdf_file)
       );
       foreach ($project->latest_release as $verdata)
       {
-        array_push(& $this->project_info["$project->projectname_short"],
+        array_push($this->project_info["$project->projectname_short"],
         $verdata->latest_release_version
                    );
```
Comment (GMishx) [inline utils/freshmeat/Classes/FreshmeatRdfs.php:158]: Can be optimized for single insertion by using array operator as bellow:
```suggestion
        $this->project_info["$project->projectname_short"][] = $verdata->latest_release_version;
```

File: utils/freshmeat/lib_projxml.h.php [hunk 2]
```diff
@@ -74,10 +74,10 @@ function get_entry($in_handle, $marker){
   while( false != ($line = fgets($in_handle, 1024))){
     // </project> is the end tag, save it and return, all done.
     if (preg_match('|</project>|', $line)){
-      array_push(&$project, $line);
+      array_push($project, $line);
```
Comment (GMishx) [inline utils/freshmeat/lib_projxml.h.php:77]: Can be optimized for single insertion:
```suggestion
      $project[] = $line;
```

File: utils/freshmeat/lib_projxml.h.php [hunk 3]
```diff
@@ -74,10 +74,10 @@ function get_entry($in_handle, $marker){
...
       return($project);
     }
-    array_push(&$project, $line);
+    array_push($project, $line);
```
Comment (GMishx) [inline utils/freshmeat/lib_projxml.h.php:80]: Can be optimized for single insertion:
```suggestion
    $project[] = $line;
```

**Review by GMishx — CHANGES_REQUESTED** [review]
Improvement suggestions.

**Comment by vargenau** [comment]
Suggestions implemented.

File: utils/freshmeat/lib_projxml.h.php (hunk 3 above)
Comment (GMishx) [reply utils/freshmeat/lib_projxml.h.php:80]: Oops, looks like you missed this suggestion :-)

**Comment by vargenau** [comment]
@GMishx
Done

**Review by GMishx — APPROVED** [review]

**Pull request merged**